x-pack/plugins/apm/dev_docs/query_debugging_in_development_and_production.md

```diff
@@ -0,0 +1,17 @@
+# Query Debugging
+
+When debugging an issue with the APM UI it can be very helpful to see the exact Elasticsearch queries and responses that was made for a given API request. 
+To enable debugging of Elasticsearch queries in APM UI do the following:
+
+1. Go to "Stack Management" 
+2. Select "Advanced Settings"
```

Comment:
Just a small suggestion, but this could be clarified a bit to save the user a little time
```suggestion
2. Under "Kibana" on the left-hand side, select "Advanced Settings"
```